wolfcrypt/src/sp_c32.c

```diff
@@ -2031,6 +2032,47 @@ static WC_INLINE sp_digit sp_2048_div_word_36(sp_digit d1, sp_digit d0,
 }
 #endif /* WOLFSSL_SP_DIV_32 */
 
+#ifndef WOLFSSL_SP_DIV_32
+static WC_INLINE sp_digit sp_2048_div_word_36(sp_digit d1, sp_digit d0,
+    sp_digit div)
+{
+#ifdef WOLFSSL_X86_64_BUILD
```

Comment:
Should the `sp_c32.c` code have this `WOLFSSL_X86_64_BUILD` block or any inline assembly? Seems like the script changes should not include that section for some platforms?

Reply:
Fixed this code to not use __divti3 on any platform and made the Intel code work for both 64 and 32 bit platforms.